README.md

```diff
@@ -1 +1,2 @@
-# code-review-project
\ No newline at end of file
+# tributepage to John Maccarthur. 
+##### in freecodecamp's project
```

Comment:
I think you should make your README.md file more elaborate so as to give further instruction on your project. For instance you could guide on how to clone and run your repo in one's local machine